models/event.js

```diff
@@ -60,7 +60,11 @@ const MODEL = {
         .example({
             nodes: [{ name: '~commit' }, { name: 'main' }, { name: 'publish' }],
             edges: [{ src: '~commit', dest: 'main' }, { src: 'main', dest: 'publish' }]
-        })
+        }),
+    pr: Joi
```

Comment:
Need to add this as an optional field in the get schema, so that the API will allow this field to be returned during a GET event request